### PR TITLE
feat: add new 1.21.2+ folders and bump pack formats

### PR DESCRIPTION
**Pull request by misode**
Adds these two folders to the resource pack

### CODE REVIEW

**Comment by RitikShah** [comment]
Running `poetry lock` should fix the `executing` 2.0.0 being yanked.

**Review by RitikShah — APPROVED** [review]

**Comment by edayot** [comment]
that odd, my PR did manage to get all dependencies and not this one
https://github.com/mcbeet/beet/pull/450

**Comment by RitikShah** [comment]
Can we get @edayot's branch into this one so there's one 1.21.2 branch that we can merge?

File: beet/library/resource_pack.py [hunk 1]
```diff
@@ -350,10 +366,12 @@ class ResourcePack(Pack[ResourcePackNamespace]):
     # fmt: off
     blockstates:      NamespaceProxyDescriptor[Blockstate]     = NamespaceProxyDescriptor(Blockstate)
     models:           NamespaceProxyDescriptor[Model]          = NamespaceProxyDescriptor(Model)
+    equipment_Models: NamespaceProxyDescriptor[EquipmentModel] = NamespaceProxyDescriptor(EquipmentModel)
```
Comment (edayot) [inline beet/library/resource_pack.py:369]: This capital M should not exist

Comment (misode) [reply beet/library/resource_pack.py:369]: We should probably call it `equipment` or `equipments` now? Since the folder moved in 1.21.4

**Comment by RitikShah** [comment]
## TODO

- [ ] Merge #450 into this (pack formats)
- [ ] `items`
- [ ] `equipments` (moved from `models/equipment`)
- [ ] Figure out `custom_model_data` merging..
  - [ ] Make it "work fine", handle special logic directly in `weld`

**Comment by misode** [comment]
@RitikShah This PR is ready now. Item model merging will have to be done in a later PR.

**Review by RitikShah — APPROVED** [review]
Changes look good, will address `beet.contrib.model_merging` in a future PR.

**Pull request merged**